### PR TITLE
test(upgrade): log more info to help debug CI flakes

### PR DESCRIPTION
**Pull request by gkalpak**
Related Jira issue: [FW-939][1]

[1]: https://angular-team.atlassian.net/browse/FW-939

### CODE REVIEW

File: packages/upgrade/test/static/angular1_providers_spec.ts [hunk 1]
```diff
@@ -22,19 +22,30 @@ import {compileFactory, injectorFactory, parseFactory, rootScopeFactory, setTemp
 
     describe('injectorFactory', () => {
       it('should return the injector value that was previously set', () => {
-        const mockInjector = {get: () => {}, has: () => false};
+        const mockInjector = {get: () => undefined, has: () => false};
```
Comment (petebacondarwin) [inline packages/upgrade/test/static/angular1_providers_spec.ts]: what difference does this change make?

**Review by petebacondarwin — APPROVED** [review]

File: packages/upgrade/test/static/angular1_providers_spec.ts (hunk 1 above)
Comment (gkalpak) [reply packages/upgrade/test/static/angular1_providers_spec.ts]: Nothing :grin: (It's just considered [good practice][1].)

[1]: https://eslint.org/docs/rules/no-empty-function

Comment (alfaproject) [reply packages/upgrade/test/static/angular1_providers_spec.ts]: As long as it doesn't get down leveled to ES5 (I know it doesn't in this case)

**Comment by AndrewKushnir** [comment]
[Presubmit](http://test/OCL:228729872:BASE:228729891:1547143617782:7e315cc0)

**Pull request closed without merging**